content/en/developers/faq/data-collection-resolution-retention.md

```diff
@@ -16,7 +16,8 @@ Find below a summary of Datadog data collection, resolution, and retention:
 | Custom Metrics (StatsD)                        | Datadog Agent (built-in statsD collector)                             | 15 seconds               | 1 second          | 15 months                                                                                                                                                    | Infrastructure       |
 | Custom Metrics (API)                           | POST directly to Datadog's API                                        | Real time                | 1 second          | 15 months                                                                                                                                                    | Infrastructure       |
 | Events                                         | Datadog Agent, integrations, or API                                   | Real time                | 1 second          | 13 months                                                                                                                                                    | Infrastructure       |
-| Traces (Sampled)                               | Datadog Agent + tracing library                                       | 10 seconds               | 1 millisecond        | 2 weeks                                                                                                                                                      | APM                  |
+| Traces (Sampled)                               | Datadog Agent + tracing library                                       | 10 seconds               | 1 millisecond        | 2 weeks (viewed traces are retained long-term, [see more here](https://docs.datadoghq.com/tracing/guide/trace_sampling_and_storage/?tab=java#trace-storage))           
+                                                           | APM                  |
```

Comment:
The new line here is causing a table formatting situation (see the preview at: https://docs-staging.datadoghq.com/morgan.lupton/trace-retention/developers/faq/data-collection-resolution-retention/ )